### DISCUSSION TITLE
post/react-query-1/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#1] [Announcements]
# post/react-query-1/

카카오페이 프론트엔드 개발자들이 React Query를 선택한 이유에 대해 설명합니다. 이 글은 연작 중 1편에 해당합니다. 1편: 카카오페이 프론트엔드 개발자들이 React Query를 선택한 이유, 2편: React Query와 함께 선언적 UI를 구성하는 방법

https://tech.kakaopay.com/post/react-query-1/

**Comment by hangooksaram** [comment]
HTTP 응답 타입 정리, 에러 핸들링 등 어떻게 통일시킬지 고민하는 시간이 너무 길어져 길을 헤멨었는데, 많은 도움이 되었습니다! 좋은 글 감사합니다!
